### PR TITLE
[new release] gr (0.0.1)

### PR DESCRIPTION
**Pull request by mseri**
OCaml bindings to the GR plotting library

- Project page: <a href="https://github.com/mseri/ocaml-gr">https://github.com/mseri/ocaml-gr</a>
- Documentation: <a href="https://mseri.github.io/ocaml-gr">https://mseri.github.io/ocaml-gr</a>

##### CHANGES:

- First release

### CODE REVIEW

File: packages/gr/gr.0.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+GR is based on an implementation of a Graphical Kernel System (GKS) and OpenGL. As a self-contained system it can quickly and easily be integrated into existing applications.
+GR is characterized by its high interoperability and can be used with modern web technologies and mobile devices."""
+maintainer: ["Marcello Seri"]
+authors: ["Marcello Seri"]
+license: "ISC"
+homepage: "https://github.com/mseri/ocaml-gr"
+bug-reports: "https://github.com/mseri/ocaml-gr/issues"
+depends: [
+  "ocaml" {>= "4.06"}
+  "dune" {> "1.11"}
```
Comment (mseri) [inline packages/gr/gr.0.0.1/opam]:
```suggestion
  "dune" {>= "1.11"}
```

File: packages/gr/gr.0.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+maintainer: ["Marcello Seri"]
+authors: ["Marcello Seri"]
+license: "ISC"
+homepage: "https://github.com/mseri/ocaml-gr"
+bug-reports: "https://github.com/mseri/ocaml-gr/issues"
+depends: [
+  "ocaml" {>= "4.06"}
+  "dune" {>= "1.11"}
+  "ctypes" {>= "0.15"}
+  "ctypes-foreign" {>= "0.15"}
```
Comment (mseri) [inline packages/gr/gr.0.0.1/opam]:
```suggestion
  "ctypes-foreign"
```

File: packages/gr/gr.0.0.1/opam [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  ["dune" "subst"] {pinned}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
```
Comment (mseri) [inline packages/gr/gr.0.0.1/opam]:
```suggestion
    "@runtest" {with-test & ocaml:version >- "4.07"}
```
I'll fix the tests upstream, there is no point in making a new release just for that

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**